rapids_build_backend/impls.py

```diff
@@ -51,6 +52,18 @@ def _get_backend(build_backend):
         )
 
 
+@lru_cache
+def _get_arch():
```

Comment:
The purpose of this function (rather than calling `platform.machine()` directly) is to make it easier to mock RBB internals.

Reply:
totally support this